common/isolationgroup/interface.go

```diff
@@ -43,5 +43,5 @@ type State interface {
 
 	// AvailableIsolationGroupsByDomainID returns the available isolation zones for a domain.
 	// Takes into account global and domain zones
-	AvailableIsolationGroupsByDomainID(ctx context.Context, domainID string, tasklistName string, availableIsolationGroups []string) (types.IsolationGroupConfiguration, error)
+	IsolationGroupsByDomainID(ctx context.Context, domainID string) (types.IsolationGroupConfiguration, error)
```

Comment:
maybe mind adding an updated comment about what the purpose of the interface is? eg "it's responsibility as a component is to return the isolation group for a domain"?